pyanaconda/modules/payloads/source/utils.py

```diff
@@ -40,3 +50,48 @@ def is_valid_install_disk(tree_dir):
     except OSError:
         pass
     return False
+
+
+def mount_simple_device_ro(device_spec, mount_point):
+    """Decode what device to mount and do so.
```

Comment:
I'm missing explanation what `simple` means in this context.

I would rather call it just `mount_ro`.

Reply:
Maybe it could be called `find_and_mount_device`?

Btw. we usually say that the device is resolved.

Reply:
Fixed.